File: CS/T548916/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace dxSample.Models
{
    public class Employee
    {
        public string Address { get; set; }
        public DateTime BirthDate { get; set; }
        public string City { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public DateTime HireDate { get; set; }
        public string HomePhone { get; set; }
        public int ID { get; set; }
        public string LastName { get; set; }
        public string Notes { get; set; }
        public string Phone { get; set; }
        public string Picture { get; internal set; }
        public string Position { get; set; }
        public string Prefix { get; set; }
        public string Skype { get; set; }
        public string State { get; set; }
        public int[] StateID { get; set; }
        public List<EmployeeTask> Tasks { get; set; }
    }
}

File: VB/T548916/Models/Employee.vb
Imports System
Imports System.Collections.Generic

Namespace T548916.Models
	Public Class Employee
		Public Property Address() As String
		Public Property BirthDate() As Date
		Public Property City() As String
		Public Property Email() As String
		Public Property FirstName() As String
		Public Property HireDate() As Date
		Public Property HomePhone() As String
		Public Property ID() As Integer
		Public Property LastName() As String
		Public Property Notes() As String
		Public Property Phone() As String
		Private privatePicture As String
		Public Property Picture() As String
			Get
				Return privatePicture
			End Get
			Friend Set(ByVal value As String)
				privatePicture = value
			End Set
		End Property
		Public Property Position() As String
		Public Property Prefix() As String
		Public Property Skype() As String
		Public Property State() As String
		Public Property StateID() As Integer()
		Public Property Tasks() As List(Of EmployeeTask)
	End Class
End Namespace